angular/projects/researchdatabox/form/src/app/component/repeatable.component.ts

```diff
@@ -108,15 +111,16 @@ export class RepeatableComponent extends FormFieldBaseComponent<Array<unknown>>
     await this.createElement(elemEntry);
   }
 
+  protected getNextLocalUniqueId(): string {
+    // Create a unique ID, appending the name, the timestamp, and a random number to ensure uniqueness.
+    return `${Date.now()}-${Math.floor(Math.random() * 10000)}`;
+  }
 
   protected createFieldNewMapEntry(templateEntry: FormFieldCompMapEntry, value: any): RepeatableElementEntry {
     // TODO: There is a potential data race condition in this method:
```

Comment:
Thanks to date.now and math.random, there's very much reduced, to no, chance for a data race.
This TODO can be removed.